accounts/utils.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+
+import requests
+from django.utils import timezone
+
+from accounts.settings import accounts_settings
+
+
+def authenticated_request(user, method, url,
```

Comment:
should probably call this `make_authenticated_request` or something? or `do_ipc` maybe? idk. but `authenticated_request` sounds like a class name rather than a function performing an action.

Reply:
In the `requests` module, you can make a request using `.request`, I was just mimicking that